### PR TITLE
feat/add-arweave

### PR DESCRIPTION
**Pull request by Kunkka0822**

### CODE REVIEW

File: src/schema.mjs [hunk 1]
```diff
@@ -157,6 +157,43 @@ export const ipfs = {
...
+    options: {
+      type: "object",
+      properties: {
+        timeout: {
+          $comment: "temporal unit is milliseconds",
+          type: "integer"
+        },
+        uri: { 
+          type: "string",
+          pattern: "ar:\/\/"
```
Comment (TimDaub) [inline src/schema.mjs:182]: is there a regex for an arweave transaction?

Comment (Kunkka0822) [reply src/schema.mjs:182]: I think it is not exact

Comment (Kunkka0822) [reply src/schema.mjs:182]: I found other regex for it.
https://github.com/ArweaveTeam/arweave-deploy/blob/0770463bff7591890376651e54093fb66f3bb487/src/commands/send.ts#L5

Comment (TimDaub) [reply src/schema.mjs:182]: great, lol I've been looking for it too. Let's try to use it, what do you think?

Comment (TimDaub) [reply src/schema.mjs:182]: why can't we just use the property name "pattern"? From my understanding that's what we do anywhere.

Comment (TimDaub) [reply src/schema.mjs:182]: ah do you need the i flag?

File: src/schema.mjs [hunk 2]
```diff
@@ -157,6 +157,43 @@ export const ipfs = {
...
+    options: {
+      type: "object",
+      properties: {
+        timeout: {
+          $comment: "temporal unit is milliseconds",
+          type: "integer"
+        },
+        uri: { 
+          type: "string",
+          pattern: "[a-zA-Z0-9-_]{43}",
```
Comment (TimDaub) [inline src/schema.mjs:182]:
```suggestion
          pattern: "ar:\/\/[a-zA-Z0-9-_]{43}.*",
```

can match
ar://ltmVC0dpe7_KxFHj0-S7mdvXSfmcJOec4_OfjwSzLRk/1/12

**Comment by il3ven** [comment]
@Kunkka0822 Arweave should be added to the `workerMessage` array.

https://github.com/neume-network/schema/blob/ea30f37863ef07e934d3036904ef471634e50462/src/schema.mjs#L212

**Comment by TimDaub** [comment]
good catch, thanks

**Pull request merged**